content/python/concepts/random-module/terms/getrandbits/getrandbits.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: .getrandbits()' 
+Description: 'Returns a pseudo-random integer with a specified number of bits.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Functions'
+  - 'Methods'
+  - 'Random'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+In the Python `random` module, the `.getrandbits()` method is used to generate a pseudo-random integer with a specified number of bits
+
+## Syntax
+
+```pseudo
+random.getrandbits(k)
+```
+
+Where `k` is the number of bits in the generated integer. The `k` must be a non-negative integer. The result will range from 0 —2<sup>k</sup> - 1.
+
+## Example
+
+In the example below, `.getrandbits()` returns an integer with 256 bits: 
+
+```py 
+import random 
+
+random_bits = random.getrandbits(256)
+
+```
```

Comment:
```suggestion
In the example below, `.getrandbits()` returns an integer with 256 bits: 

```py 
import random 

random_bits = random.getrandbits(256)

```
```
I would suggest you add another code that properly demonstrates the use of  `getrandbits` in Python. Also, add s shell block to show the output of the code.